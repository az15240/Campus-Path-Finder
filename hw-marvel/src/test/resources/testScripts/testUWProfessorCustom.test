# Tests data set from UWProfessorCustom.csv. Adapted from https://edstem.org/us/courses/32028/discussion/2601400.
# This file mainly tests the basic actions. Paths are short ones with maximum of 2 edges.

LoadGraph g UWProfessorsCustom.csv

ListNodes g

ListChildren g Perkins

FindPath g Perkins Weber

FindPath g Perkins Grossman

FindPath g Perkins J.Lee

FindPath g xxx yyy