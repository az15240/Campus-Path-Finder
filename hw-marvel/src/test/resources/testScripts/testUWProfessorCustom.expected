# Tests data set from UWProfessorCustom.csv. Adapted from https://edstem.org/us/courses/32028/discussion/2601400.
# This file mainly tests the basic actions. Paths are short ones with maximum of 2 edges.

loaded graph g

g contains: A.Rao Beame Grossman Hsia J.Lee Karlin Kohlbrenner Kohno Leang Maas OveisGharan Perkins R.Lin Ri.Anderson Roesner Ru.Anderson Schafer Tang Tessaro Thachuk Tsun Weber Wilcox Y.Lee Zatloukal

the children of Perkins in g are: Hsia(333) Thachuk(333) Wilcox(331) Zatloukal(331)

path from Perkins to Weber:
Perkins to Zatloukal via 331
Zatloukal to Weber via 311

path from Perkins to Grossman:
Perkins to Wilcox via 331
Wilcox to Grossman via 341

path from Perkins to J.Lee:
no path found

unknown: xxx
unknown: yyy